roles/v2k-bridge/templates/v2k_bridge_cluster.yaml.j2

```diff
@@ -19,6 +19,15 @@ spec:
       containers:
       - name: v2k-bridge
         image: dojot/v2k-bridge:{{ dojot_v2k_version}}
+        readinessProbe:
```

Comment:
And about livenessProbe and startupProbe?

Reply:
Startup Probe is used by services that take too long to start, which is not this case.
The liveness I don't know if would be too useful, since NodeJS crashes the process when it encounters any unhandled promise or something else, thus leaving the `restartPolicy` to do the job.